auto_submit/lib/service/revert_review_template.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// A template for creating follow up review issues for created revert requests.
+class RevertReviewTemplate {
+  RevertReviewTemplate({
+    required this.repositorySlug,
+    required this.revertPrNumber,
+    required this.revertPrAuthor,
+    required this.originalPrLink,
+  });
+
+  final String repositorySlug;
+  final int revertPrNumber;
+  final String revertPrAuthor;
+  final String originalPrLink;
+
+  /// Constructs the issues title.
+  String _constructTitle() {
+    return '''
+Review request for Revert PR $repositorySlug#$revertPrNumber
+''';
+  }
+
+  /// Constructs the issues body.
+  String _constructBody() {
+    return '''
+Pull request $repositorySlug#$revertPrNumber was submitted and merged by
+$revertPrAuthor in order to revert changes made in this pull request $originalPrLink.
+
+Please assign this issue to the person that will make the formal review on the
+revert request listed above.
+
+Please do the following so that we may track this issue to completion:
+1. Add the reviewer of revert pull request as the assignee of this issue.
+2. Add the label 'revert_review' to this issue.
+3. Close only when the review has been completed.
+
+
+PLEASE DO NOT MODIFY THE FOLLOWING
```

Comment:
You can help users do the right thing here, and make it a comment (so it's hidden unless edited). app_dart already does this to check if it's already commented on flaky issues.

```
DO NOT EDIT, REVERT METADATA
<!--
{
  // jsonData Here
}
-->
```

Reply:
Oh man I did not know this. Thank you! I will go ahead and make that change.

Reply:
Done.